docs/samples/browser-plugin-meetings/app.js

```diff
@@ -347,8 +349,13 @@ function createMeeting(e) {
   webex.meetings.create(value, type)
     .then((meeting) => {
       createMeetingDestinationElm.value = '';
+      displayMeetingStatusElm.innerHTML = '';
       generalStartReceivingTranscription.disabled = false; // eslint-disable-line no-use-before-define
       refreshMeetings();
+    }).catch((error) => {
+      if(error.code === 101){
+        displayMeetingStatusElm.innerHTML = spaceIDError;
```

Comment:
We should actually display the error that is passed from the SDK so we can update the integration tests in the future accordingly.

Reply:
@mkesavan13, we don't want to display all the errors, only displaying the error related to spaceID, Also Error needs formatting in the sample.

Reply:
Error code 30105 added as Space ID deprecated as of beta, Updated file packages/@webex/plugin-meetings/src/common/errors/webex-errors.ts
cc - @mkesavan13, @arun3528